content/rendering-with-batch/aws-fis/fis-resilience-create.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
We have to start this section what an AWS FIS experiment is, and then relating it to the configuration of the compute environment